vernac/comDefinition.ml

```diff
@@ -32,9 +32,11 @@ let check_imps ~impsty ~impsbody =
   match impsty, impsbody with
   | a1 :: impsty, a2 :: impsbody ->
     (match a1.CAst.v, a2.CAst.v with
-    | None , None -> aux impsty impsbody
-    | Some _ , Some _ -> aux impsty impsbody
-    | _, _ -> warn_implicits_in_term ?loc:a2.CAst.loc ())
+    | None , None | Some _, None -> aux impsty impsbody
+    | Some (_,b1) , Some (_,b2) ->
```

Comment:
We could also compare the names (to fail eg `Definition foo : forall {A}, A -> A := fun {B} b => b.`) but not sure if useful.